scipy/linalg/tests/test_lapack.py

```diff
@@ -138,6 +138,66 @@ def test_clapack(self):
 
 class TestLeastSquaresSolvers(TestCase):
 
+    def test_gels(self):
+        for dtype in REAL_DTYPES:
+            a1 = np.array([[1.0,2.0],
+                          [4.0,5.0],
+                          [7.0,8.0]], dtype=dtype)
+            b1 = np.array([16.0, 17.0, 20.0], dtype=dtype)
+            gels, gels_lwork = get_lapack_funcs(('gels','gels_lwork'),
+                                                (a1, b1))
+
+            m, n = a1.shape
+            if len(b1.shape) == 2:
+                nrhs = b1.shape[1]
+            else:
+                nrhs = 1
+
+            # Request of sizes
+            work, info = gels_lwork(m,n,nrhs)
```

Comment:
Minor comment: I'd maybe rewrite this to check for `lwork = _compute_lwork(gels_lwork, m, n, nrhs)` where that routine comes from `from scipy.linalg.lapack import _compute_lwork`